src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingPixels.java

```diff
@@ -89,6 +96,23 @@ public void adsTrackingPixelSentCuratedMainPages(String wiki, String page, Strin
     assertTrackingPixelsSent(adsBaseObject, pixelUrls);
   }
 
+  @NetworkTrafficDump
+  @Test(
+          groups = "AdsTrackingPixelsAuthPage",
+          dataProviderClass = AdsDataProvider.class,
+          dataProvider = "adsTrackingPixelsSentAuthPage"
+  )
+  @InBrowser(
+          browser = Browser.CHROME,
+          emulator = Emulator.GOOGLE_NEXUS_5
+  )
+  public void adsTrackingPixelSentAuthPage(String wiki, String page, String[] pixelUrls) {
```

Comment:
It's not only about number of lines. It's more about better maintenance in future. Currently if we change a way we verify  code works we'd need to change two methods instead of one.
When it comes to naming, what do you think about `verifyAdsTrackingPixelSent` ?